### PR TITLE
added more activations

### PR DESCRIPTION
**Pull request by AI-WAIFU**

### CODE REVIEW

**Comment by ClashLuke** [comment]
I'd propose to add square and perhaps even cube but remove sign (as it has no gradient). What do you think?

File: models/gpt2/gpt2.py [hunk 1]
```diff
@@ -275,6 +275,25 @@ def get_activation_fn(params):
...
+    elif activation_fn == "sin":
+        return mtf.sin
+    elif activation_fn == "cos":
+        return mtf.cos
+    elif activation_fn == "sign":
+        return mtf.sign
+    elif activation_fn == "triangle relax":
+        return lambda x: mtf.sin(x)-mtf.sin(3*x)/9+mtf.sin(5*x)/25-mtf.sin(7*x)/49
+    elif activation_fn == "square relax":
+        return lambda x: mtf.cos(x)-mtf.cos(3*x)/3+mtf.cos(5*x)/5-mtf.cos(7*x)/7
```
Comment (leogao2) [inline models/gpt2/gpt2.py:292]: no spaces in names

**Review by leogao2 — CHANGES_REQUESTED** [review]

**Comment by leogao2** [comment]
Also, I'm not going to test sign and id in the actual ablation because I don't think that would be very informative at all

File: models/gpt2/gpt2.py (hunk 1 above)
Comment (ClashLuke) [reply models/gpt2/gpt2.py:292]: what about underscores, capitalization, or dashes? should the names really be `thisisaverylongname`?

Comment (leogao2) [reply models/gpt2/gpt2.py:292]: underscores are the preferred way, spaces are best avoided because they wreak all sorts of havoc.

**Review by leogao2 — APPROVED** [review]

**Pull request merged**